BlueprintUI/Sources/Layout/EqualSizeStack.swift

```diff
@@ -0,0 +1,134 @@
+/// An element that sizes all of its children equally, stacking them according to
+/// the specified `direction` and spacing them according to the specified `gutter`.
+public struct EqualSizeStack: Element {
+
+    /// The direction in which this element will stack its children.
+    public var direction: Direction
+
+    /// The amount of space between children in this element.
+    public var gutter: CGFloat
+
+    /// The child elements to be laid out.
+    public var children: [Element]
+
+    /// - Parameters:
+    ///     - direction: The direction in which this element will stack its children.
+    ///     - gutter: The amount of space between children in this element. Defaults to 0.
+    ///     - children: The child elements to be laid out. Defaults to an empty array.
+    ///     - configure: A closure allowing the element to be further customized. Defaults to a no-op.
+    public init(
+        direction: Direction,
+        gutter: CGFloat = 0,
+        children: [Element] = [],
+        configure: (inout EqualSizeStack) -> Void = { _ in })
```

Comment:
Is the configure block useful if the initializer already provides all values to be customized?

Reply:
not really; can remove.

Reply:
It’s useful for adding children, just like how `Row` and `Column` do:

```swift
column.add(
    child: EqualSizeStack(
        direction: .horizontal,
        spacing: 16,
        configure: { stack in
            stack.add(child: Box(backgroundColor: .red))
            stack.add(child: Box(backgroundColor: .blue))
        }))
```

Reply:
Ah yeah, true. I guess having both the `children` and this threw me off

Reply:
(I had posted this, but my comment disappeared?)

Ah yeah. I guess having both the `children` argument and this threw me off, but that makes total sense.

Reply:
Oh I totally missed the `children` parameter 🤦‍♂. I guess maybe this doesn’t as much need the `configure` since it has no traits. Maybe just keep give one way or the other (array or `configure`).

Reply:
If picking one, I would probably lean towards the `configure` route for consistency with Row & Column. The `add(child:)` API also provides an opportunity for keyed children.

Reply:
+1